axlearn_inference.py

```diff
@@ -352,6 +386,7 @@ def pad_list(l, max_len, fill_value=tokenizer.pad_id):
     llama_probs = torch.softmax(output.logits, dim=-1).numpy()
```

Comment:
Would like a test where we get the softmax output from axlearn directly.
We should check that the logits used for that softmax input AND output is also in FP32.
It is ok to PR to an accuracy branch of axlearn to check the softmax input AND output dtype.

Reply:
will work on it